### PR TITLE
Namespace for TZ cronjobber

### PR DESCRIPTION
**Pull request by razvan-moj**
**Why**
Requested in https://github.com/ministryofjustice/cloud-platform/issues/804

### CODE REVIEW

**Comment by razvan-moj** [comment]
tzupdater is not needed, our nodes have /usr/share/zoneinfo/* from the Debian package

**Comment by digitalronin** [comment]
Please could you add a note in the user guide explaining that this option for user cron jobs exists, and where to find the details?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/cronjobber/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (alkar) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/cronjobber/build.sh]: I think we should have this as a separate job in concourse (building the image). Right now I'm assuming the intention is to build manually whenever we need to?

**Comment by razvan-moj** [comment]
> Please could you add a note in the user guide explaining that this option for user cron jobs exists, and where to find the details?

Sure, docs will be a separate PR in the other repo

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/cronjobber/build.sh (hunk 1 above)
Comment (razvan-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/cronjobber/build.sh]: kube-system is a small change so sure, will check Concourse too and amend

**Pull request closed without merging**